src/app/components/icons/IconDrawer.tsx

```diff
@@ -77,6 +93,11 @@ export const IconDrawer: React.FC = () => {
         dispatch({ type: TOGGLE_SIDEBAR, payload: false });
     };
 
+    useEffect(() => {
+        setIconSize(24 as IconSize);
+        setIconColor('Black' as IconColor);
```

Comment:
only change these if we go from a material icon to a pxblue icon. You can check this with the usePrevious hook (should already be available in doc-it, I think):

```
const previousSelectedIcon = usePrevious(selectedIcon);
...
useEffect(() => {
    if(previousSelectedIcon && previousSelectedIcon.isMaterial !== selectedIcon.isMaterial){
        { /* reset the size */ }
    }
}, [selectedIcon);
```